fml/ascii_trie.cc

```diff
@@ -0,0 +1,68 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+#include "flutter/fml/ascii_trie.h"
+
+namespace fml {
+typedef AsciiTrie::TrieNode TrieNode;
+
+namespace {
+void Add(TrieNode** trie, const char* entry) {
+  int ch = entry[0];
+  assert(ch < 128);
```

Comment:
Here and elsewhere, please stick to the standard FML utilities for assertion and logging.

Reply:
done